### PR TITLE
i10n

### PR DESCRIPTION
**Pull request by ovari**
i10n quotation marks
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table

install.rdf adds translations in localized versions so indicate all languages which have been translated.
If you know the translator for a localization, please add.

Removed as requested:
<s>Quicktext version bump to 1.2
When released on GitHub will show as newer than the current ATN version of 1.1.
https://github.com/thundernest/quicktext/releases
https://addons.thunderbird.net/addon/quicktext/
</s>

Thank you

### CODE REVIEW

File: chrome/locale/en-US/quicktext.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@ template=Template
 newGroup=New group
 newTemplate=New template
 newScript=New script
-remove=Are you sure you want to remove "%S"?
+remove=Are you sure you want to remove “%S”?
```
Comment (jobisoft) [inline chrome/locale/en-US/quicktext.properties:14]: you use differnt quotes for differnt languages (sometimes at the top, sometimes atthe bottom). Where is that defined?

File: chrome/locale/sv-SE/quicktext.properties [hunk 2]
```diff
@@ -11,9 +11,9 @@ template=Mall
 newGroup=Ny grupp
 newTemplate=Ny mall
 newScript=Nytt skript
-remove=Är du säker på att du vill tabort "%S"?
+remove=Är du säker på att du vill tabort ”%S”?
```
Comment (jobisoft) [inline chrome/locale/sv-SE/quicktext.properties:14]: the same for opening and closing quote?

File: install.rdf [hunk 3]
```diff
@@ -8,11 +8,17 @@
...
-    <em:translator>Óvári (hu)</em:translator>    
+    <em:contributor>John Bieling</em:contributor>
+    <em:contributor>Philippe Lieser</em:contributor>
+    <em:translator>(de)</em:translator>
+    <em:translator>Emil Hesslow (en-US)</em:translator>
+    <em:translator>(fr)</em:translator>
+    <em:translator>Óvári (hu)</em:translator>
+    <em:translator>(ja)</em:translator>
+    <em:translator>(sv-SE)</em:translator>
+    <em:creator>Emil Hesslow</em:creator>
```
Comment (jobisoft) [inline install.rdf]: is there a reason, you moved the creator to the bottom?

File: install.rdf [hunk 4]
```diff
@@ -26,15 +32,93 @@
...
+        <em:description>Adds a toolbar with unlimited number of text to quickly insert. It’s also possible to use variables like [[TO=firstname]]. With settings for everything.</em:description>
+        <em:contributor>R Kent James</em:contributor>
+        <em:contributor>John Bieling</em:contributor>
+        <em:contributor>Philippe Lieser</em:contributor>
+        <em:translator>(de)</em:translator>
+        <em:translator>Emil Hesslow (en-US)</em:translator>
+        <em:translator>(fr)</em:translator>
         <em:translator>Óvári (hu)</em:translator>
+        <em:translator>(ja)</em:translator>
+        <em:translator>(sv-SE)</em:translator>
```
Comment (jobisoft) [inline install.rdf]: I do not know any of the missing translator, please remove placeholders

**Review by jobisoft — CHANGES_REQUESTED** [review]

File: chrome/locale/en-US/quicktext.properties (hunk 1 above)
Comment (ovari) [reply chrome/locale/en-US/quicktext.properties:14]: i10n quotation marks
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table

File: chrome/locale/sv-SE/quicktext.properties (hunk 2 above)
Comment (ovari) [reply chrome/locale/sv-SE/quicktext.properties:14]: i10n quotation marks
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table

File: install.rdf (hunk 3 above)
Comment (ovari) [reply install.rdf]: Copied what you did in your TBSync add-on

File: install.rdf (hunk 4 above)
Comment (ovari) [reply install.rdf]: Do you you prefer `Anonymous (ja)` like the [Mail Merge add-on](https://addons.thunderbird.net/addon/mail-merge/) as shown in the image below?

kep

Comment (jobisoft) [reply install.rdf]: I would go for Unknown, what do you think?

File: install.rdf (hunk 3 above)
Comment (jobisoft) [reply install.rdf]: But this is Emil, I would like to have him at the top as before.

File: chrome/locale/sv-SE/quicktext.properties (hunk 2 above)
Comment (jobisoft) [reply chrome/locale/sv-SE/quicktext.properties:14]: Thanks for the link.

File: install.rdf (hunk 3 above)
Comment (ovari) [reply install.rdf]: done

File: install.rdf (hunk 4 above)
Comment (ovari) [reply install.rdf]: Done

**Review by jobisoft — APPROVED** [review]

**Comment by jobisoft** [comment]
Thanks!

**Comment by ovari** [comment]
You’re welcome

**Pull request merged**